src/cookies.md.njk

```diff
@@ -1,7 +1,118 @@
 ---
 title: Cookies
-layout: layout-archived.njk
-ignore_in_sitemap: true
+description: GOV.UK Design System puts small files (known as ‘cookies’) on your computer
+layout: layout-single-page-prose.njk
 ---
 
-For up to date information, see [the privacy policy](/privacy-policy/).
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+
+# Cookies
+
+[GOV.UK Design System](https://design-system.service.gov.uk/) puts small files (known as ‘cookies’) on your computer.
+
+These cookies are used across the GOV.UK Design System, which also includes
+GOV.UK Frontend and Prototype Kit websites. Your preferences will apply to all
```

Comment:
This doesn't apply at the moment - I think we'd want to leave this out until we actually add sub-domain consent.